src/metabase/db/data_migrations.clj

```diff
@@ -180,7 +180,7 @@
 ;; continue to run as legacy SQL.
 (defmigration ^{:author "camsaul", :added "0.30.0"} add-legacy-sql-directive-to-bigquery-sql-cards
   ;; For each BigQuery database...
-  (doseq [database-id (db/select-ids Database :engine "bigquery")]
+  (doseq [database-id (db/select-ids Database :engine "bigquery")] ; TODO: keep this?
```

Comment:
We should go ahead and take this out now. Since we'd be upgrading to the new driver before we ever reach this point this migration will effectively always be a no-op. 

If anyone migrates from 29 or older to 43 or newer they'll have to manually add `#legacy` directives to their BigQuery SQL queries written in legacy SQL

Reply:
Done.